sdk/src/main/java/co/omise/android/models/SourceType.kt

```diff
@@ -72,6 +72,8 @@ sealed class SourceType(
 
     object PayPay : SourceType("paypay")
 
+    object WeChatPay : SourceType("wechat_pay")
```

Comment:
@AnasNaouchi we used different capitalization scheme for Java. Which is correct? https://github.com/omise/omise-java/pull/164/files

Reply:
`WeChatPay` is the correct one. Will change in omise-java